errors/errors_test.go

```diff
@@ -54,7 +54,7 @@ func TestContextOnGoErrors(t *testing.T) {
 }
 
 func TestContextOnWrappedErrors(t *testing.T) {
-	err := NewFatalError(errors.New("Go error"))
+	err := NewFatalError(errors.New("go error"))
```

Comment:
I think it was intended in this file for these to refer to the Go language, in which case an upper case first letter would be appropriate here.

Reply:
But at the same time this file says `t.Error("go error should not be a fatal error")` even before my changes.